src/server/game/Entities/Player/Player.cpp

```diff
@@ -23358,7 +23358,17 @@ bool Player::IsSpellFitByClassAndRace(uint32 spell_id) const
 
     SkillLineAbilityMapBounds bounds = sSpellMgr->GetSkillLineAbilityMapBounds(spell_id);
     if (bounds.first == bounds.second)
+    {
+        if (const SpellInfo* spellInfo = sSpellMgr->GetSpellInfo(spell_id))
+        {
+            if (ChrClassesEntry const* classEntry = sChrClassesStore.LookupEntry(getClass()))
```

Comment:
getClassMask is called earlier, and class gets checked on player Load. Is it necessary to check ChrClassesEntry against null?